MAUI/Scheduler/cell-selection-view.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: Cell selection view in .NET MAUI Scheduler | Syncfusion
```

Comment:
Cell selection in the .NET MAUI Scheduler | Syncfusion

Reply:
Changes made sir